test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -124,11 +127,11 @@ string TestFunctionCall::format(
 
 		/// Format either the expected output or the actual result output
 		string result;
-		if (!_renderResult)
+		if (_renderMode != RenderMode::ActualValuesExpectedGas)
 		{
 			bool const isFailure = m_call.expectations.failure;
 			result = isFailure ?
-				formatFailure(_errorReporter, m_call, m_rawBytes, _renderResult, highlight) :
+				formatFailure(_errorReporter, m_call, m_rawBytes, _renderMode == RenderMode::ActualValuesExpectedGas, highlight) :
```

Comment:
This always evaluates to `false` in this branch of the loop:
```suggestion
				formatFailure(_errorReporter, m_call, m_rawBytes, /* _renderResult */ false, highlight) :
```